src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -89,28 +90,47 @@
 			@lightbox-closed="filtersVisible = false"
 		>
 			<div class="loan-filter-controls">
-				<gender-filter
-					:initial-gender="initialGender"
-					:selected-gender="selectedGender"
-					@updated-filters="handleUpdatedFilters"
-				/>
-
-				<kv-accordion-item
-					class="loan-filters__lightbox-accordian"
-					id="sort-order-accordian"
+				<span
+					class="tw-flex tw-items-center"
```

Comment:
I think you'll need to add `tw-grid-cols-2` here so that the 2 contained divs below are spaced evenly within this container.

Reply:
You'll prob want to remove `tw-items-center` as that is likely causing the top header of each section to no line up